### PR TITLE
Service operations #41

### PR DESCRIPTION
**Pull request by FJMentz**
This is NOT a FINAL VERSION. Just want to push to dev so that networks and volumes have a template to work from and also so that I can get some of develop's stuff ;).

### CODE REVIEW

**Comment by FJMentz** [comment]
@CDuPlooy Please help 😭

File: src/app/_shared/services/list-view/service-list-view.component.html [hunk 1]
```diff
@@ -1,9 +1,30 @@
-<!--TODO(FJMentz): Do date stuff (commented)-->
-
 <div id="serviceList">
     <div *ngFor="let service of services; let i = index">
         <div class="card ServiceListCard">
             <div class="card-header" [attr.id]="'heading-'+ i" data-toggle="collapse" [attr.data-target]="'#collapse-' + i" aria-expanded="true" [attr.aria-controls]="'collapse-' + i">
+<<<<<<< HEAD
```
Comment (egeldenhuys) [inline src/app/_shared/services/list-view/service-list-view.component.html]: Merge conflict here

File: src/app/_shared/services/list-view/service-list-view.component.html [hunk 2]
```diff
@@ -27,6 +49,14 @@ <h5 class="mb-0">
 
                     <li class="list-group-item d-flex justify-content-between sub-item">
                         Replicas: {{ service.Spec.Mode.Replicated.Replicas}}
+<<<<<<< HEAD
```
Comment (egeldenhuys) [inline src/app/_shared/services/list-view/service-list-view.component.html]: More here

**Review by egeldenhuys — CHANGES_REQUESTED** [review]

File: src/app/_shared/services/list-view/service-list-view.component.html (hunk 1 above)
Comment (FJMentz) [reply src/app/_shared/services/list-view/service-list-view.component.html]: haha saw it just now, pushing as we speak

**Comment by FJMentz** [comment]
I'm pretty sure I got them all now.

**Review by egeldenhuys — APPROVED** [review]

**Review by CDuPlooy — APPROVED** [review]

**Review by annamarieHelberg — APPROVED** [review]
Good enough! Thank you!

**Review by ghost — APPROVED** [review]

**Pull request merged**